tests/80torture/03events.pl

```diff
@@ -54,3 +54,32 @@
          params => { from => $user->eventstream_token, timeout => "hello" },
       )->main::expect_http_4xx;
    };
+
+test "Event size limits",
+   requires => do {
+      my $user = local_user_preparer();
+      [ $user, room_preparer( requires_users => [ $user ] ) ];
```

Comment:
FYI, there's `local_user_and_room_preparers` in `10apidoc/33room-members.pl` which does exactly this. You could use that instead.